akvo/rsr/tests/test_permissions.py

```diff
@@ -192,6 +192,9 @@ def test_org_user_permissions(self):
         # None object permissions
         self.assertTrue(user.has_perm('rsr.view_project', None))
 
+        # Test no permissions on None, if user doesn't have required employment
+        self.assertFalse(user.has_perm('rsr.add_indicatorperioddata'))
```

Comment:
This test fails when it's called from test_org_enumerator_permissions() 😢

Reply:
That is the intended behaviour.  As I understand it, checking for a permission on None, is essentially only checking if the user has an employment of the given role, or not.  And, enumerators have the permission to add indicator period updates.

Reply:
Before the fix, the predicates were not checking if the user had the given kind of employment or not, and this was the bug the commit tries to fix.

Reply:
Ah, sorry, I didn't realize the test was calling another test. Bad test habits, I guess. I'll fix the test.